doc/source/examples/incompressible-flow/2d-taylor-couette-flow-nitsche/2d-taylor-couette-flow-nitsche.rst

```diff
@@ -0,0 +1,358 @@
+========================================================
+2D Taylor-Couette Flow Using Nitsche Immersed Boundaries
+========================================================
+
+This is the XXe Lethe example. It revisits the same taylor-couette flow problem in `Example 2 <https://lethe-cfd.github.io/lethe/examples/incompressible-flow/2d-taylor-couette-flow/2d-taylor-couette-flow.html>`, 
+now using immersed boundaries to represent the inner cylinder. This example demonstrates some of the capabilities of Lethe to simulate the flow around complex geometries 
+ meshing them explicitly with a conformal mesh, but instead by using the Nitsche immersed boundary method available within Lethe.
```

Comment:
The skipped line seems to create a box in the doc, is it supposed to be like this?
![image](https://user-images.githubusercontent.com/50414572/150169761-38947c17-6a66-42d3-b0a1-c121ca0e1c20.png)

Reply:
Thanks!